admin/languages.php

```diff
@@ -176,6 +177,145 @@
                                 '" . zen_db_input($ezpage['pages_html_text']) . "')");
         }
 
+// create additional products option stock names records
+        $products_option_stock_names = $db->Execute("SELECT pos_name_id, pos_name
+                                       FROM " . TABLE_PRODUCTS_OPTIONS_STOCK_NAMES . "
+                                       WHERE language_id = " . (int)$_SESSION['languages_id']);
+
+        foreach ($products_option_stock_names as $option_stock_name) {
+          $db->Execute("INSERT INTO " . TABLE_PRODUCTS_OPTIONS_STOCK_NAMES . " (pos_name_id, language_id, pos_name)
+                        VALUES ('" . (int)$option_stock_name['pos_name_id'] . "',
+                                '" . (int)$insert_id . "',
+                                '" . zen_db_input($option_stock_name['pos_name']) . "')");
+        }
```

Comment:
And this POSM fix, as another separate PR, could probably be merged by @lat9 quickly as well.

Reply:
Yes but I realized there is a problem here. If POSM had never been installed, constant TABLE_PRODUCTS_OPTIONS_STOCK_NAMES does not exists.
A test must be added before running this code like this:
`if (defined('TABLE_PRODUCTS_OPTIONS_STOCK_NAMES')) { ...}`
I will submit the corrected version.